src/StatusQ/Components/StatusChatList.qml

```diff
@@ -96,31 +94,27 @@ Column {
                 StatusChatListItem {
 
                     id: statusChatListItem
-
                     property string profileImage: ""
-
                     opacity: dragSensor.active ? 0.0 : 1.0
                     Component.onCompleted: {
                         if (typeof statusChatList.profileImageFn === "function") {
                             profileImage = statusChatList.profileImageFn(model.chatId || model.id) || ""
                         }
                     }
                     originalOrder: model.position
-                    chatId: model.chatId || model.id
-                    categoryId: model.categoryId || ""
+                    chatId: model.itemId || model.id
+                    categoryId: model.itemId || ""
                     name: !!statusChatList.chatNameFn ? statusChatList.chatNameFn(model) : model.name
-                    type: model.chatType
+                    type: !!model.type ? model.type : StatusChatListItem.Type.CommunityChat
                     muted: !!model.muted
-                    hasUnreadMessages: !!model.hasUnreadMessages || model.unviewedMessagesCount > 0
-                    hasMention: model.mentionsCount > 0
-                    badge.value: model.chatType === StatusChatListItem.Type.OneToOneChat ?
-                        model.unviewedMessagesCount || 0 :
-                        model.mentionsCount || 0
-                    selected: (model.chatId || model.id) === statusChatList.selectedChatId
+                    hasUnreadMessages: !!model.hasNotification || model.unviewedMessagesCount > 0
+                    hasMention: model.notificationsCount > 0
+                    badge.value: model.notificationsCount
+                    selected: (model.itemId || model.id) === statusChatList.selectedChatId
```

Comment:
@alexandraB99 there is `active` field maintained from the backend so this should be just `selected: model.active`

Reply:
done